src/Microsoft.Health.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs

```diff
@@ -64,7 +65,7 @@ public AuditEventTypeMappingTests()
 
             _auditEventTypeMapping = new AuditEventTypeMapping(_actionDescriptorCollectionProvider);
 
-            ((IStartable)_auditEventTypeMapping).Start();
+            ((IHostedService)_auditEventTypeMapping).StartAsync(CancellationToken.None);
```

Comment:
Ideally we would await for a async task to finish, unless it is fire and forget. Should we move this out of the constructor?

Reply:
Done.